web/apps/apps_test.go

```diff
@@ -653,14 +656,24 @@ func TestApps(t *testing.T) {
 		token, err := testInstance.MakeJWT(consts.AccessTokenAudience, flagship.ClientID, "*", "", time.Now())
 		require.NoError(t, err)
 
+		// Send logs for a client side konnector
+		e.POST("/konnectors/"+clientSideSlug+"/logs").
+			WithHost(testInstance.Domain).
+			WithHeader("Authorization", "Bearer "+token).
+			WithBytes([]byte(`[ { "timestamp": "2022-10-27T17:13:38.382Z", "level": "error", "msg": "This is an error message" } ]`)).
+			Expect().Status(204)
+
+		assert.Equal(t, `time="2022-10-27T17:13:38.382Z" level=error msg="This is an error message" domain=`+domain+" job_id= nspace=jobs slug="+clientSideSlug+" worker_id=client\n", testOutput.String())
+
 		// Send logs for a konnector
-		e.POST("/konnectors/"+slug+"/logs").
+		testOutput.Reset()
```

Comment:
@nono I'm not sure if this one is still needed?

Reply:
It is needed, we are using `testOutput` in the previous block of code, and if we don't `Reset` it, we will still have its log message in `testOutput` making the test fail.